src/main/resources/schema.sql

```diff
@@ -58,23 +58,36 @@ CREATE TABLE user_details
     blood_type CHAR(3),
     PRIMARY KEY (id)
 );
+CREATE TABLE requester_details
+(
+    id INT NOT NULL AUTO_INCREMENT,
+    first_name VARCHAR(100) NOT NULL,
+    last_name VARCHAR(100) NOT NULL,
+    email VARCHAR(320) NOT NULL,
+    phone_number VARCHAR(11),
+    blood_type CHAR(3),
+    hospital_name VARCHAR(100) NOT NULL,
+    street_name VARCHAR(100),
+    city_name VARCHAR(100),
+    state_name VARCHAR(100),
+    zip_code VARCHAR(10),
+    message VARCHAR(500),
+    PRIMARY KEY (id)
+);
 
-ALTER TABLE user_details ADD CONSTRAINT fk_user_details_accounts
-   FOREIGN KEY (id) REFERENCES accounts(id)
-   ON DELETE CASCADE
-   ON UPDATE CASCADE;
-
-CREATE TABLE feedback(
+CREATE TABLE requester_time
+(
     id INT NOT NULL AUTO_INCREMENT,
-    message TEXT NOT NULL,
-    account_id INT NOT NULL,
+    email VARCHAR(320) NOT NULL,
+    time_requested VARCHAR(100),
     PRIMARY KEY (id)
 );
 
-ALTER TABLE feedback ADD CONSTRAINT fk_feedback_account_id

```

Comment:
For some reason the feedback table is not in the schema.sql file anymore. This is causing the unit tests to break for all the repositories.